index.html

```diff
@@ -491,64 +485,80 @@ <h2>
+      errors or user/programmatic cancellation.
     </p>
     <p>
-      A [=credential request coordinator=] has an associated [=credential
-      request client=], which it interacts with.
+      The [=credential request coordinator=]:
     </p>
+    <ul>
+      <li>Validates and transforms presentation or issuance inputs and outputs.
+      </li>
+      <li>Interprets each individual request in the input list as targeting a
+      specific [=holder=], and prepares [=holder=] options accordingly.
+      </li>
+      <li>Presents the available [=holders=] to the user for selection.
+      </li>
+      <li>Resolves the interaction's {{Promise}} with the selected [=digital
+      credential=] or rejects it to indicate cancellation.
+      </li>
+    </ul>
+    <aside class="note">
+      <p>
+        Although the coordinator handles input/output coordination, the
+        user-selected [=holder=] typically acts as the [=credential chooser=].
+      </p>
+    </aside>
     <h3>
       Interaction states
     </h3>
     <p>
       The [=credential request coordinator=] has a finite set of
       <dfn data-dfn-for="credential request coordinator">interaction
-      states</dfn>, which are used to manage the lifecycle of a request for a
-      digital credential, whether it is a [=digital credential/presentation
-      request=] or an [=digital credential/issuance request=]. The states are:
+      states</dfn> used to manage the lifecycle of a [=digital
+      credential/presentation request=] or [=digital credential/issuance
+      request=]:
     </p>
     <dl>
       <dt>
-        <dfn data-dfn-for="credential request coordinator">Idle</dfn>:
+        "<dfn data-dfn-for="credential request coordinator">idle</dfn>":
       </dt>
       <dd>
         No credential interaction is currently in progress.
       </dd>
       <dt>
-        <dfn data-dfn-for="credential request coordinator">Requesting</dfn>:
+       "<dfn data-dfn-for="credential request coordinator">requesting</dfn>":
       </dt>
       <dd>
-        A credential interaction is currently in progress and the user
-        interface is being presented to the user.
+        A credential interaction is in progress and the user interface is
+        presented.
       </dd>
       <dt>
-        <dfn data-dfn-for="credential request coordinator">Aborting</dfn>:
+        "<dfn data-dfn-for="credential request coordinator">aborting</dfn>":
       </dt>
       <dd>
-        A credential interaction is being canceled, because of an error, a user
-        cancellation, or a [=AbortController/signal abort=]. The coordinator is
-        in the process of cleaning up the interaction in order to return to the
-        [=credential request coordinator/idle=] state.
+        The active interaction is being canceled due to an error, user action,
+        or an [=AbortController/signal abort=]; the coordinator is cleaning up
+        before returning to [=credential request coordinator/idle=].
```

Comment:
Either all of the causes of cancellation should have an indefinite article (`a`/`an`), or only the first should. I think all is better. Also, `an` should not precede consonant-sounding words (e.g., `signal`), only vowel-sounding word (e.g., `abort`).
```suggestion
        The active interaction is being canceled due to an error, a user action,
        or a [=AbortController/signal abort=]; the coordinator is cleaning up
        before returning to [=credential request coordinator/idle=].
```